tests/test_storage_macro.rs

```diff
@@ -27,6 +27,16 @@ async fn test_storage_macro_redis() -> Result<(), Box<dyn std::error::Error + Se
     Ok(())
 }
 
+#[tokio::test]
+async fn test_storage_macro_fs() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    let (_ir, _is, cl) = set_up_test_env_single_user().await?;
+
+    let key_name = "storage_macro_test_fs:$fs:/tmp/colink-sm-fs-test";
```

Comment:
wait. I thought it was based on the value of the stored key?
e.g. value of `storage_macro_test_fs`: `/tmp/colink-sm-fs-test`, then
- read `storage_macro_test_fs:$fs"` -> get content of `/tmp/colink-sm-fs-test`

Personally, I think this might be more general as it is similar to mounting the fs to our key-value storage. Or if you think otherwise, we can have a discussion.